File: algorithms/algo-method/Swift/2-10.swift
// https://algo-method.com/tasks/33

func main() {
    let s = readLine()!
    let nm = readLine()!.split(separator: " ").compactMap { Int($0) }
    let n = nm[0]
    let m = nm[1]
    
    var sArray = Array(s)
    sArray.swapAt(n-1, m-1)

    let output = String(sArray)
    print(output)
}

main()

File: algorithms/algo-method/Kotlin/2-10.kt
// https://algo-method.com/tasks/33

fun main() {
    var s = readLine()!!.toMutableList()
    val (n, m) = readLine()!!.split(" ").map{ it.toInt() }

    s[n-1] = s[m-1].also { s[m-1] = s[n-1]} 

    println(s.joinToString(""))
}

// fun main() {
//     var s = readLine()!!.toMutableList()
//     val (n, m) = readLine()!!.split(" ").map{ it.toInt() }

//     val temp = s[n-1]
//     s[n-1] = s[m-1]
//     s[m-1] = temp

//     println(s.joinToString(""))
// }
